app/Http/Controllers/AppointmentController.php

```diff
@@ -18,6 +17,17 @@ public function __construct(AppointmentService $service)
         $this->service = $service;
     }
 
+    public function index()
+    {
+        $appointments = $this->service->index();
+        return response()->json(IndexAppointmentResource::collection($appointments), Response::HTTP_OK);
+    }
+
+    public function approve()
+    {
+        $this->service->approve();
+        return response()->json([], Response::HTTP_OK);
```

Comment:
Send the appointment ID from the controller to the service. The service should not get any data directly from the request so it become reusable.

Reply:
done